unpackaged/test_packaged/relationship_autocreate/classes/REL_Relationships_Con_PTST.cls

```diff
@@ -28,38 +28,37 @@
     POSSIBILITY OF SUCH DAMAGE.
 */
 /**
-* @author Salesforce.org
-* @date 2020
-* @group Relationships
-* @description Unmanaged tests for REL_Relationships_Con_TDTM to verify 
-* auto creation functionality works as expected with multiple mappings.
-*/
+ * @author Salesforce.org
+ * @date 2020
+ * @group Relationships
+ * @description Unmanaged tests for REL_Relationships_Con_TDTM to verify
+ * auto creation functionality works as expected with multiple mappings.
+ */
 @isTest
 private class REL_Relationships_Con_PTST {
-
     private static Integer loopCount = 250;
 
     /*********************************************************************************************************
-    * @description Unpackaged test to verify that when Relationship Auto Creation is enabled, and multiple 
-    * mappings exist, inserting a Contact record with all Auto Creation mapped fields populated generates 
-    * relationship  and reciprocal relationship records for each mapped field.
-    *********************************************************************************************************/
+     * @description Unpackaged test to verify that when Relationship Auto Creation is enabled, and multiple
+     * mappings exist, inserting a Contact record with all Auto Creation mapped fields populated generates
+     * relationship  and reciprocal relationship records for each mapped field.
+     *********************************************************************************************************/
     @isTest
-    private static void insertContactWithMultipleAutoCreateMappingsAllPopulated(){
-        REL_Relationships_Con_AfterInsert_MTST.setupMultipleAutoCreateRelationships();
+    private static void insertContactWithMultipleAutoCreateMappingsAllPopulated() {
+        REL_Relationships_Con_PTST.setupMultipleAutoCreateRelationships();
```

Comment:
This is the first actual code change so the code references the appropriate class